File: Atividade 3/ESTRUTURA-DE-DADOS-atividade2/src/aluno.cs

public class Aluno{
    
    public string nome;
    public int idade;
    public float nota;
    
    public Aluno(string _nome, int _idade, float _nota){
        nome = _nome;
        idade = _idade;
        nota = _nota;
    }
    
    public string Nome {
        get {
            return nome;
        }
        set => nome = value;
        
    }
    
    public int Idade {
        get {
            return idade;
        }
        set => idade = value;
    }
    
    public float Nota{
        get {
            return nota;
        }
        set => nota = value;
    }
    
}

File: Prova 2/Mapas/Aluno.java
public class Aluno {
    private int matricula;
    private String nome;
    private int idade;

    public Aluno(int matricula, String nome, int idade){
        this.matricula = matricula;
        this.nome = nome;
        this.idade = idade;

    }

    public int getMatricula(){
        return matricula;
    }

    public String getNome(){
        return nome;
    }

    public void setNome(String nome){
        this.nome = nome;
    }

    public int getIdade(){
        return idade;
    }

    public void setIdade(int idade){
        this.idade = idade;
    }

    public String toString(){
        return "Nome: "+nome+" Idade: "+idade+" Matricula: "+matricula+"."; 
    }
}
